tests/fixtures.py

```diff
@@ -61,6 +62,10 @@ def attributes_for_round(id=0, **kwargs):
     }
 
 
+def attributes_for_chat_room(id=0, **kwargs):
+    return {**kwargs}
```

Comment:
`attributes_for_chat_room`

If you are just passing the kwargs, there's no reason why you want to define the fn right